templates/default/onlinefs-kafka.properties.erb

```diff
@@ -1,10 +1,10 @@
 bootstrap.servers = <%= @kafka_fqdn %>:<%= node['kkafka']['broker']['port'] %>
 security.protocol = SSL
 ssl.truststore.location = /certs/<%= node['onlinefs']['user'] %>__tstore.jks
-ssl.truststore.password = adminpw
+ssl.truststore.password = <%= node['kkafka']['broker']['ssl']['truststore']['password']%>
 ssl.keystore.location = /certs/<%= node['onlinefs']['user'] %>__kstore.jks
-ssl.keystore.password = adminpw
-ssl.key.password = adminpw
+ssl.keystore.password = <%= node['kkafka']['consumer']['ssl']['keystore']['password']%>
```

Comment:
Use `broker` everywhere

Reply:
good point, I guess have missed it when copying, fixed it now